bin/resources/GameIndex.yaml

```diff
@@ -50677,8 +50677,8 @@ TCES-52033:
       content: |-
         author=YukiXXL
         // Speed Correction (25 FPS)
-        patch=1,EE,00175848,extended,00000019
         patch=1,EE,00175a7c,extended,00000019
+        patch=1,EE,00175848,extended,00000019
```

Comment:
any reason you swapped these lines around? they were already in the right order.

Reply:
yes, the game has 3 addresses, one of them is ProScan so it runs 30 FPS and we don't need a patch for it, the other 2 are PAL addresses and one of them is PAL address no.1 which is active when you boot the game and the 2nd is PAL address no.2 which becomes active after you change to ProScan and change back to Normal mode, so the patch was placed as PAL no.2 above PAL no.1 i merely changed it to the correct order, simple change for a cleaner code, same effect tho, just wanna be precise.

Reply:
You realise the order of the patch lines has no determination on when it will fire, right? both lines are evaluated/patched every single vsync from boot, not at specific points, if you want to do that you'd need to do more advanced patching, but I'm presuming it doesn't matter.

the first number `patch=1` can be 0, 1 or 2, with 0 being on boot only, 1 being every vsync, and 2 being on boot and every vsync, but tbh just leaving it as it is should be fine, I just didn't see the point in changing the order, all it does is hurts my OCD seeing the addresses out of order :P

Reply:
exactly like i said the effect is the same, it was hurting my OCD as i know PAL address no.1 should be first so i put it above

Reply:
*sigh* fine.